### DISCUSSION TITLE
AI 技術助你清潔口腔

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#225] [科技新知]
AI現在真的是無所不在，Colgate（高露潔）推出Hum智能電動牙刷內置感應器，能夠追蹤用戶的刷牙時間、頻率和覆蓋範圍，資料可以連接到手機上的應用程式，並再配合 3D 聲波振動技術鎖定需要進一步清潔的特定部位。
https://chinese.engadget.com/electric-tooth-brush-deals-20-020036479.html

**Comment by 107602505** [comment]
這個只能電動牙刷內置感應器對於有戴牙套的人來說可能是一大福音，因為戴牙套常常有很多地方都刷不太到，若能透過這個追蹤刷牙的頻率和覆蓋範圍對保養口腔健康會是很大的幫助!

**Comment by JAIMEHUNG** [comment]
感覺讓刷牙變得很有趣XD但能有效幫助到刷牙常刷不乾淨的人！

**Comment by HSU-YU-JEN** [comment]
這個技術超棒，應該可以解決很多牙痛問題

**Comment by luo910227** [comment]
特價才台幣一千多塊，對我這個有點強迫症的人實在是一大福音，有錢可以當盤子的時候可以考慮買買。AI技術真的無所不在，只差在想不想得到能夠怎樣運用而已。

**Comment by 109802503gloria** [comment]
這個技術看起來主要靠偵測位置和刷牙時間來推斷有沒有刷乾淨，感覺還是不能做到告訴你細部的牙菌斑位置，可能會變成有花很多時間刷，但都沒刷到該刷的地方，我覺得牙菌斑顯色劑還是比較靠譜。

**Comment by ChinChun0220** [comment]
應該還無法非常準確知道位置，沒辦法像掃地機器人那樣，但是個很有趣的應用XD

**Comment by zixunn** [comment]
這幾年我家長輩有陸續使用電動牙刷進行口腔清潔，電動牙刷內置感應器確實非常前衛哈哈，但感覺潔牙仍要依照自己的狀況並對著鏡子刷，清潔才能確實到位。

**Comment by chupochun** [comment]
感覺這樣的產品一定價值不斐，但是若能真正有效清潔口腔的話，消費者應該會買單的

**Comment by ghost** [comment]
突然想到前陣子個根管回憶.....你各位要好好保護牙齒阿

**Comment by d26488399** [comment]
我覺得這項技術能有效預防小孩的蛀牙，很棒

**Comment by yucheng1028** [comment]
當醫療衛生遇上科技，總是會帶來許多的便利，讚讚

**Comment by Yuhsin000** [comment]
希望之後有直接幫我把牙刷乾淨的技術

**Comment by xkllkx** [comment]
沒想到AI已經那麼普及到日常生活了，謝謝樓主分享

**Comment by pekoranoinu** [comment]
希望這個發明能夠在台灣上市，來讓我買一個試試。

**Comment by bloodtrail0511** [comment]
如果這個產品早一點問世，我可能就可以少去找牙醫一點了QQ

**Comment by PoooYen** [comment]
覺得高露潔可以發明牙膏或漱口水裡面塞奈米機器人，然後漱口五分鐘以內就由機器人去口腔部位裡面進行特定區域清理以及美白，費用的話每一次就收一萬就好。

**Comment by Sazmafuji** [comment]
希望此技術更加發展這樣就不用再因為蛀牙去看牙醫

**Comment by sean1325418** [comment]
感覺以後所有生活用品都會連上網，AI會記錄所有事

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，AI真的是未來的趨勢。

**Comment by hapiwangy** [comment]
這類型的科技對於不太清楚要特別注重哪裡清潔的人來說是非常方便的!!
